From 1bb02b98cfedfd6b51ac8de0c34a5778659433f7 Mon Sep 17 00:00:00 2001
From: Arsen Arsen <arsenarsentmc@outlook.com>
Date: Thu, 6 Sep 2018 10:29:09 +0200
Subject: [Media Player] Change int to double for positions

In previous versions of Plasma, media player had an overflow bug in positions, leading to the player resetting to 00:00.
This only affects files longer than around 33 minutes 20 seconds.

BUG: 397591
FIXED-IN: 5.12.7

Differential Revision: https://phabricator.kde.org/D15311
---
 applets/mediacontroller/contents/ui/ExpandedRepresentation.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/applets/mediacontroller/contents/ui/ExpandedRepresentation.qml b/applets/mediacontroller/contents/ui/ExpandedRepresentation.qml
index 2922f99..904fe49 100644
--- a/applets/mediacontroller/contents/ui/ExpandedRepresentation.qml
+++ b/applets/mediacontroller/contents/ui/ExpandedRepresentation.qml
@@ -35,7 +35,7 @@ Item {
 
     readonly property int controlSize: Math.min(height, width) / 4
 
-    property int position: mpris2Source.currentData.Position || 0
+    property double position: mpris2Source.currentData.Position || 0
     readonly property real rate: mpris2Source.currentData.Rate || 1
     readonly property double length: currentMetadata ? currentMetadata["mpris:length"] || 0 : 0
 
-- 
cgit v0.11.2
